exe/Main.hs

```diff
@@ -24,6 +53,11 @@ cliParser = do
   content <- NE.some (argument str (metavar "SOURCES..."))
   output <- strOption (long "output" <> short 'o' <> metavar "OUTPUT" <> value "tailwind.css")
   mode <- modeParser
+  bPluginTypography  <- switch (long "plugin-typography"   <> help "enable official typography plugin")
+  bPluginForms       <- switch (long "plugin-forms"        <> help "enable official forms plugin")
+  bPluginLineClamp   <- switch (long "plugin-line-clamp"   <> help "enable official line-clamp plugin")
+  bPluginAspectRatio <- switch (long "plugin-aspect-ratio" <> help "enable official aspect ratio plugin")
+  bAllPlugins        <- switch (long "all-plugins" <> short 'a' <> help "enable all official plugins")
```

Comment:
The new CLI disables all plugins by default, no? Existing behaviour (all plugins enabled) shouldn't change.

Also, I feel like we want to have one option that takes comma-separated values: `--plugins="typography,forms,line-clamp,aspect-ratio"`.

Reply:
> The new CLI disables all plugins by default, no? Existing behaviour (all plugins enabled) shouldn't change.

got it

> Also, I feel like we want to have one option that takes comma-separated values: --plugins="typography,forms,line-clamp,aspect-ratio".

Yes, I can certainly do that. There isn't a standard way of passing list-like stuff via command-line, unfortunately.
The interface that you suggest is very clear. It requires custom reader instances for `data Plugin` and will bloat the code a bit more.